auth-web/src/components/pay/eft/ShortNameTransactions.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <BaseVDataTable
+    id="eft-transactions-table"
+    class="transaction-list"
+    itemKey="id"
+    :loading="state.loading"
+    loadingText="Loading Transaction Records..."
+    noDataText="No Transaction Records"
+    :setItems="state.results"
+    :setHeaders="headers"
+    :setTableDataOptions="state.options"
+    :totalItems="state.totalResults"
+    :filters="state.filters"
+    :pageHide="true"
+    :hideFilters="true"
+    :hasTitleSlot="true"
+    :useObserver="true"
+    :observerCallback="infiniteScrollCallback"
+    :height="'400px'"
```

Comment:
height="400px" I think works too

Reply:
Base table seems to require a string or it errors

Reply:
weird that is a string